value.c

```diff
@@ -240,6 +240,186 @@ bool parse_uservalue_float(const char *nptr, uservalue_t * val)
     return true;
 }
 
+#define DEFAULT_UINTLS_SZ (1000)
+
+/* XXX: this should be in a utility file of some sort (no pun intended) */
+static int _uint64_cmp(const void *i1, const void *i2)
+{
+    return (*(const uint64_t *)i1 - *(const uint64_t *)i2);
+}
```

Comment:
I imagine this works because it's implicitly converted to `int` back at the function return.

Is this 100% guaranteed to work in all cases?